src/Sylius/Bundle/AdminBundle/Resources/config/form/events.yml

```
@@ -0,0 +1,302 @@
+                    priority: 10
+        sylius.admin.shipment.ship_with_tracking_code_content:
+            blocks:
+                content:
+                    template: '@SyliusAdmin/Shipment/Grid/Action/_shipWithTrackingCodeContent.html.twig'
+                    priority: 10
+        sylius.admin.shipment.grid.ship_content:
+            blocks:
+                content:
+                    template: '@SyliusAdmin/Shipment/Grid/_shipContent.html.twig'
+                    priority: 10
+        sylius.admin.shipment.partial.ship_content:
+            blocks:
+                content:
+                    template: '@SyliusAdmin/Shipment/Partial/_shipContent.html.twig'
+                    priority: 10
+        sylius.admin.shipment.show.content:
+            blocks:
+                header:
+                    template: '@SyliusAdmin/Shipment/Show/_headerContent.html.twig'
+                    priority: 30
+                content:
+                    template: '@SyliusAdmin/Shipment/Show/_breadcrumb.html.twig'
+                    priority: 20
+                shipment_units:
+                    template: '@SyliusAdmin/Shipment/Show/_shipmentUnits.html.twig'
+                    priority: 10
+        sylius.admin.shipment.show.breadcrumb:
+            blocks:
+                content:
+                    template: '@SyliusAdmin/Shipment/Show/_breadcrumb.html.twig'
+                    priority: 10
+        sylius.admin.order.show.header_items:
+            blocks:
+                content:
+                    template: '@SyliusAdmin/Order/Show/_headerItems.html.twig'
+                    priority: 10
+        sylius.admin.order.show.payment_content:
+            blocks:
+                header:
+                    template: '@SyliusAdmin/Order/Show/Payment/_header.html.twig'
+                    priority: 40
+                content:
+                    template: '@SyliusAdmin/Order/Show/Payment/_content.html.twig'
+                    priority: 30
+                transitions_complete:
+                    template: '@SyliusAdmin/Order/Show/Payment/_stateMachineComplete.html.twig'
+                    priority: 20
+                transitions_refund:
+                    template: '@SyliusAdmin/Order/Show/Payment/_stateMachineRefund.html.twig'
+                    priority: 10
+        sylius.admin.order.show.shipment_content:
+            blocks:
+                header:
+                    template: '@SyliusAdmin/Order/Show/Shipment/_header.html.twig'
+                    priority: 60
+                content:
+                    template: '@SyliusAdmin/Order/Show/Shipment/_content.html.twig'
+                    priority: 50
+                transitions:
+                    template: '@SyliusAdmin/Order/Show/Shipment/_stateMachineShip.html.twig'
+                    priority: 40
+                shipment_tracking:
+                    template: '@SyliusAdmin/Order/Show/Shipment/_shipmentTracking.html.twig'
```

```suggestion
                    template: '@SyliusAdmin/Order/Show/Shipment/_tracking.html.twig'
```
and adjust block name